ethcore/src/client/client.rs

```diff
@@ -298,7 +297,7 @@ impl Importer {
 
 						let transactions_len = closed_block.transactions().len();
 
-						let route = self.commit_block(closed_block, &header, &bytes, client);
+						let route = self.commit_block(closed_block, &header, encoded::Block::new(bytes), client);
```

Comment:
in future, I'd like this function (and blockchain.rs functions) to take as argument some `MetaBlock` that contains both plain block and it's encoded version. Currently we are encoding and then decoding stuff that we just encoded. That's not efficient